vMenuServer/config/languages.json

```diff
@@ -0,0 +1,1631 @@
+{
```

Comment:
Hate to say this, but please use something similar to the permissions, like `save_loadout` instead of `Save Loadout` for the keys, that way if I ever change any English text, nobody has to go through their languages.json and rename everything that has changed whenever an update is released.